widgetssdk/src/main/java/com/glia/widgets/chat/ChatActivity.java

```diff
@@ -68,13 +75,32 @@ public static Intent getIntent(
 
     /**
      * Creates and fills out Intent for starting ChatActivity
+     * @param context   - Context object
+     * @param contextId - Context asset ID
```

Comment:
What do you think about changing it a bit to distinguish Android and Glia Context:
 ```
* @param context   - Android Context object
 * @param contextId - Glia visitor context asset ID
```